h2o-core/src/main/java/water/server/H2oServletContainerLoader.java

```diff
@@ -0,0 +1,28 @@
+package water.server;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+
+/**
+ * Finds implementation of {@link H2oServletContainerFacade} found on the classpath.
+ * There must be exactly one present.
+ */
+public class H2oServletContainerLoader {
+  public static final H2oServletContainerFacade INSTANCE;
+  static {
+    final ServiceLoader<H2oServletContainerFacade> serviceLoader = ServiceLoader.load(H2oServletContainerFacade.class);
+    final Iterator<H2oServletContainerFacade> iter = serviceLoader.iterator();
+    if (! iter.hasNext()) {
+      throw new IllegalStateException("No implementation of H2oServletContainerFacade found on classpath");
```

Comment:
Maybe we should also check for the case when we find more implementations?

Reply:
we do, right below this, and we fail with explanatory message